<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EONPC-18" ignored="0" name="FirefoxTest" tests="14" failures="2" timestamp="2019-10-09T13:10:09 IST" time="197.01" errors="0">
  <testcase name="emptyCompanyNameTest" time="9.164" classname="com.lambdatest.SignUpTest">
    <failure type="java.lang.AssertionError" message="expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]">
      <![CDATA[java.lang.AssertionError: expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]
at com.lambdatest.SignUpTest.emptyCompanyNameTest(SignUpTest.java:235)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- emptyCompanyNameTest -->
  <testcase name="emptyEmailTest" time="9.297" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyNameTest" time="9.508" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyPasswordTest" time="9.626" classname="com.lambdatest.SignUpTest"/>
  <testcase name="emptyPhoneTest" time="9.674" classname="com.lambdatest.SignUpTest"/>
  <testcase name="inValidPasswordTest" time="9.233" classname="com.lambdatest.SignUpTest"/>
  <testcase name="inValidPhoneTest" time="9.55" classname="com.lambdatest.SignUpTest"/>
  <testcase name="invalidEmailTest" time="9.119" classname="com.lambdatest.SignUpTest"/>
  <testcase name="loginRedirectionTest" time="2.441" classname="com.lambdatest.SignUpTest"/>
  <testcase name="privacyPolicyRedirectionTest" time="3.511" classname="com.lambdatest.SignUpTest"/>
  <testcase name="termsRedirectionTest" time="4.159" classname="com.lambdatest.SignUpTest"/>
  <testcase name="uncheckedTerms" time="8.079" classname="com.lambdatest.SignUpTest"/>
  <testcase name="validRegistrationTest" time="8.729" classname="com.lambdatest.SignUpTest">
    <failure type="java.lang.AssertionError" message="expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]">
      <![CDATA[java.lang.AssertionError: expected [https://accounts.lambdatest.com/email/verify] but found [https://accounts.lambdatest.com/register]
at com.lambdatest.SignUpTest.validRegistrationTest(SignUpTest.java:192)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- validRegistrationTest -->
  <testcase name="verifyElemntsOnPageTest" time="4.405" classname="com.lambdatest.SignUpTest"/>
</testsuite> <!-- FirefoxTest -->
